support/batch_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package support
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestGetJob(t *testing.T) {
+
+	g := gomega.NewGomegaWithT(t)
```

Comment:
Do you need gomega in here? IMHO all usage can be replaced by `test` which you declare below.

Reply:
Gotcha we don't need it. Yes I replace it with test as well as changed in ingress_test.go